docs/examples/MarrmotM14.ipynb

```diff
@@ -2,204 +2,144 @@
  "cells": [
```

Comment:
<p>Typo, replace <code>uip</code>  with <code>up</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>it is fixed.</p>